core/rfb.js

```diff
@@ -863,6 +864,10 @@ export default class RFB extends EventTargetMixin {
                 this._rfb_auth_scheme = 16; // Tight
             } else if (includes(2, types)) {
                 this._rfb_auth_scheme = 2; // VNC Auth
+            } else if (includes(17, types)) {
+                this._rfb_auth_scheme = 17; // UltraVNC
+            } else if (includes(104, types)) {
+                this._rfb_auth_scheme = 104; // UltraVNC SC
```

Comment:
Note that this list sets which authentication forms we prefer. Are you sure these belong last?

Reply:
This is the authentication process required by UltraVNC's Signle Click mode.

Reply:
My point was, do you really want the other authentication types to be used instead of single click?

How does single click work from a user's point of view?